index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compactible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./styles.css" />
+    </head>
+    <body>
+        <header class="header">
+            <h2 class="logo">PETER</h2>
+    <nav class="navbar">
+        <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd"
+                d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                fill="#acb7c3" />
+        </svg>              
+    </nav>
+    </header>
+            <div class="top-section">
+                <section>
+                        <h2 class="title">Hi there.<br>I'm Peter</h2>
```

Comment:
- [ ] This is the main title of your page. The proper tag for it should be `h1`. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info about that. Kindly change it to `h1`.
- [ ] It is required to stick to the guiding design as much as possible. This includes font-size, font-family, color, background-color, margin, padding among others. The font-family for the `h1` should `Crete Round` as it is presented on the figma design. Kindly import that font from [google font](https://fonts.google.com/about) and use it for your main title.